tests/telemetry/test_telemetry.py

```diff
@@ -45,6 +54,21 @@ def setup_telemetry_forpyclient(duthost):
     else:
         logger.info('client auth is false. No need to restart telemetry')
 
+def generate_client_cli(duthost, method=METHOD_GET, xpath="COUNTERS/Ethernet0", target="COUNTERS_DB", subscribe_mode=SUBSCRIBE_MODE_STREAM, submode=SUBMODE_SAMPLE, intervalms=0, update_count=3):
+    """Generate the py_gnmicli command line based on the given params.
+    """
+    cmdFormat = 'python /gnxi/gnmi_cli_py/py_gnmicli.py -g -t {0} -p {1} -m {2} -x {3} -xt {4} -o {5}'
+    cmd = cmdFormat.format(duthost.mgmt_ip, TELEMETRY_PORT, method, xpath, target, "ndastreamingservertest")
+
+    if method == METHOD_SUBSCRIBE:
```

Comment:
Is this for future purpose? I don't see call for METHOD_SUBSCRIBE  anywhere?

Reply:
Yes, I have another PR for testing sampled streaming that uses this constant. This part of the code is common among them.

https://github.com/Azure/sonic-mgmt/pull/2660